tests/test_glue/test_glue.py

```diff
@@ -107,6 +110,9 @@ def test_get_job_exists():
     job.should.have.key("SecurityConfiguration")
     job.should.have.key("NotificationProperty")
     job.should.have.key("GlueVersion")
+    job.should.have.key("CodeGenConfigurationNodes")
```

Comment:
Minor nitpick - could you rewrite these to use traditional asserts? 
`assert x in job`

We've decided to move away from sure, the library that allows `.should.have`-notation, because it makes debugging more difficult than it needs to be. 
This will be a gradual process, but we're starting with ensuring that new PR's do not use it.

Reply:
@bblommers sure, I changed to assert in the next commit. Please let me know if that's enough or I should change more in this file.